### DISCUSSION TITLE
In a "match by dragging" test, is it possible to have more questions than answers?

### DISCUSSION THREAD

**Discussion by javierlabanca** [chamilo/chamilo-lms#5398] [General] [answered]
Hi! How is everybody?

I was wondering if it was possible to configure the "match by dragging" test so that the options on the left are more in number than the options on the right. I'm attaching an image as an example. This would be very useful to drag more than one item from the left into the same category on the right, Please see what I mean by looking at the attached image.

![match by dragging](https://github.com/chamilo/chamilo-lms/assets/3823430/1135c49c-3a30-4621-8cae-e5e72315d2ce)

In the example attached, the students would have to match two words from the left to the "nouns" option on the right, And two words from the left to the "adjectives" option on the right.

As you can see, options C and D on the right are not necessary and shouldn't be displayed.

Regards,
Javier

**Comment by LudiscapeXApi** [comment, marked as answer]
Hi,

Hello, No, it's not possible on Chamilo.

**Reply by javierlabanca** [reply]
Thank you for the answer!
